### ISSUE TITLE
NTR, pulmo: pulmonary fissures

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5588]
**Preferred term label:**
pulmonary fissures

**Synonyms**

**Definition (free text, please give PubMed ID)**
A fissure is the infolding of visceral pleura that separates one lobe or part of a lobe from another; thus, the interlobar fissures are produced by two layers of visceral pleura. Supernumerary fissures usually separate segments rather than lobes. The azygos fissure, unlike the other fissures, is formed by two layers each of visceral and parietal pleura. All fissures (apart from the azygos fissure) may be incomplete. Fissures appear in CT images as linear opacities, normally 1 mm or less in thickness, that correspond in position and extent to the anatomic fissural separation of pulmonary lobes or segments. Qualifiers include minor, major, horizontal, oblique, accessory, anomalous, azygos, and inferior accessory.

PMID: 18195376

**Parent term (use hpo.jax.org/app)**
Abnormal of the pleura, HP0002103

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU: crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
@mgriese, @katrinknoflach

Do we mean "Supernumerary pulmonary fissure" as the actual abnormality/label of this term?

**Comment by GrieseAG** [comment]
No, there are three different kind of fissures, as described in the definition: 
Interlobar fissures (produced by two layers of visceral pleura)
Supernumerary fissures (separate segments rather than lobes)
Azygos fissure (formed by two layers each of visceral and parietal pleura)

All need to be differentiated, but commonly it is searched for interlobar fissures, first.

**Comment by pnrobinson** [comment]
Discussion as follows. Todo -- definition

Abnormal pulmonary fissure morphology
--Abnormal pulmonary fissure architecture
--Abnormal pulmonary fissure count
----increased pulmonary fissure count
----reduced pulmonary fissure count

**Comment by pnrobinson** [comment]
Suggested definition
Abnormal pulmonary fissure morphology
def: An abnormal form or number of the pulmonary fissures. A fissure is the infolding of visceral pleura that separates one lobe or part of a lobe from another; thus, the interlobar fissures are produced by two layers of visceral pleura. The azygos fissure, unlike the other fissures, is formed by two layers each of visceral and parietal pleura. Supernumerary fissures usually separate segments rather than lobes. Supernumerary fissures usually separate segments rather than lobes.

**Comment by pnrobinson** [comment]
@GrieseAG  Adding
Supernumerary pulmonary fissure syn increased pulmonary fissure count
def: Presence of a lung fissure that does not exist normally. Supernumerary fissures include the superior accessory fissure, the medial basal fissure, the left horizontal fissure, and the azygos fissure form supernumerary lobes.

**Comment by pnrobinson** [comment]
Adding 
Decreased pulmonary fissure count
Lack of one or more of the normal pulmonary fissures.

@GrieseAG   - please reopen if one of the above definitions needs more work!
